site/content/community/_index.md

```diff
@@ -14,9 +14,10 @@ You can follow the work we do, see our milestones, and our backlog on our [GitHu
 * Follow us on Twitter at [@projectvelero](https://twitter.com/projectvelero)
 * Join our Kubernetes Slack channel and talk to over 800 other community members: [#velero](https://kubernetes.slack.com/messages/velero)
 * Join our [Google Group](https://groups.google.com/forum/#!forum/projectvelero) to get updates on the project and invites to community meetings.
-* Join the Velero community meetings  - [Zoom link](https://VMware.zoom.us/j/94501971662?pwd=aUxVbWVEWHZSbDh4ZGdGU1cxYUFoZz09):
-  * 1st and 3rd Tuesday at 12PM ET / 9AM PT ([Convert to your time zone](https://dateful.com/convert/est-edt-eastern-time?t=12pm))
-  * 2nd and 4th Wednesday at 8am China Standard Time / Tuesday 7pm EST (8pm EDT) / Tuesday 4pm PST (5pm PDT) ([Convert to your time zone](https://dateful.com/convert/beijing-china?t=8am))
-  * Read and comment on the [meeting notes](https://hackmd.io/Jq6F5zqZR7S80CeDWUklkA?view)
-  * See previous community meetings on our [YouTube Channel](https://www.youtube.com/playlist?list=PL7bmigfV0EqQRysvqvqOtRNk4L5S7uqwM)
-  * Have a question to discuss in the community meeting? Please add it to our [Q&A Discussion board](https://github.com/vmware-tanzu/velero/discussions/categories/community-support-q-a)
+* Join the Velero community meetings  - [Zoom link](https://VMware.zoom.us/j/94501971662?pwd=aUxVbWVEWHZSbDh4ZGdGU1cxYUFoZz09)  
+Bi-weekly  community meeting alternating every week between Beijing Friendly timezone and EST/Europe Friendly Timezone  
+  * Beijing/US friendly - we start at 8am Beijing Time(CST) / 8pm EDT(7pm EST) / 5pm PDT(4pm PST) / 1am CET(2am CEST) - [Convert to your time zone](https://dateful.com/convert/beijing-china?t=8am))  
+  * US/Europe friendly - we start at 10pm Beijing Time (CST) / 10am EDT(9am EST) / 7am PDT(6am PST) / 3pm CET(4pm CEST)  - [Convert to your time zone](https://dateful.com/convert/est-edt-eastern-time?t=10))  
```

Comment:
So this one isn't quite correct. Since the US/Europe-friendly meeting is scheduled based on US timezones, the Eastern/Pactific/Europe times don't change throughout the year -- should be 10 AM/7 AM/4 PM all year long. However, the Beijing time switches when the United States goes on winter/standard time. Basically the meeting is at 10pm Beijing Time (CST) while the US is on daylight/summer time, but it moves to 11pm CST when the US is on standard/winter time.